### PR TITLE
refactor: reorg get user information section

### PR DESCRIPTION
**Pull request by xiaoyijun**
## Summary

Split the original "Scopes and claims" section into to"Get user info APIs" and  "Scopes and claims" and update some "scopes and claims" section to "Get user information"

image

image

### Test

Go through all SDK pages and everything work well.

### CODE REVIEW

**Review by wangsijie — APPROVED** [review]

File: docs/sdk/fragments/_get-user-information.mdx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import ScopeClaimList from './_scope-claim-list.md';
+
+{props.getIdTokenClaimsApi && props.fetchUserInfoApi &&
+
+<>
+You can use Logto API to retrieve user information:
+
+<ul>
+  <li>
+    <code>{props.getIdTokenClaimsApi}</code>: get user information from ID token.
```
Comment (simeng-li) [inline docs/sdk/fragments/_get-user-information.mdx:10]: getIdTokenClaims is not an API? Also might double check all the SDKs provides that method

Comment (simeng-li) [reply docs/sdk/fragments/_get-user-information.mdx:10]: Also just like sign-in sign-out, should we provide sample code as props here? 
calling getIdTokenClaimsn and getUserInfo using different SDK.

File: docs/sdk/fragments/_get-user-information.mdx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  </li>
+  <li>
+    <code>{props.fetchUserInfoApi}</code>: fetch user information from the OIDC{' '}
+    <a href="https://openid.net/specs/openid-connect-core-1_0.html#UserInfo">Userinfo Endpoint</a>{' '}
+    provided by Logto.
+  </li>
+</ul>
+
+<p>
+  It's important to note that the user information claims that can be retrieved depend on the scopes
```
Comment (charIeszhao) [inline docs/sdk/fragments/_get-user-information.mdx:20]:
```suggestion
  It's important to note that the user information claims that can be retrieved depending on the scopes
```

File: docs/sdk/fragments/_get-user-information.mdx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  used by the user during signing-in, and considering performance and data size, the ID token may
```
Comment (charIeszhao) [inline docs/sdk/fragments/_get-user-information.mdx:21]:
```suggestion
  used by the user during signing-in. And considering performance and data size, the ID token may
```

File: docs/sdk/fragments/_get-user-information.mdx [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  not contain all user claims, some user claims only available in the userinfo endpoint (see the
```
Comment (charIeszhao) [inline docs/sdk/fragments/_get-user-information.mdx:22]:
```suggestion
  not contain all user claims, some user claims are only available in the userinfo endpoint (see the
```

File: docs/sdk/fragments/_get-user-information.mdx [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+  related list below).
+</p>
+
+</>}
+
+Logto uses OIDC [scopes and claims conventions](https://openid.net/specs/openid-connect-core-1_0.html#Claims) to define the scopes and claims for retrieving user information from the ID token and OIDC <a href="https://openid.net/specs/openid-connect-core-1_0.html#UserInfo">Userinfo Endpoint</a>. Both of "scope" and "claim" are terms from the OAuth 2.0 and OpenID Connect (OIDC) specifications.
```
Comment (charIeszhao) [inline docs/sdk/fragments/_get-user-information.mdx:28]: nit:
```suggestion
Logto uses OIDC [scopes and claims conventions](https://openid.net/specs/openid-connect-core-1_0.html#Claims) to define the scopes and claims for retrieving user information from the ID token and OIDC <a href="https://openid.net/specs/openid-connect-core-1_0.html#UserInfo">Userinfo Endpoint</a>. Both of the "scope" and the "claim" are terms from the OAuth 2.0 and OpenID Connect (OIDC) specifications.
```

File: docs/sdk/fragments/_get-user-information.mdx [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+In short, when you request a scope, you will get the corresponding claims in the user information. For example, if you request the `email` scope, you will get the `email` and `email_verified` data of the user.
+
+By default, Logto SDK will always requests three scopes: `openid`, `profile`, and `offline_access`, and there is no way to remove these default scopes. But you can add more scopes when configuring the Logto:
```
Comment (charIeszhao) [inline docs/sdk/fragments/_get-user-information.mdx:32]:
```suggestion
By default, Logto SDK will always request three scopes: `openid`, `profile`, and `offline_access`, and there is no way to remove these default scopes. But you can add more scopes when configuring Logto:
```

**Review by charIeszhao — APPROVED** [review]

File: docs/sdk/fragments/_get-user-information.mdx (hunk 1 above)
Comment (xiaoyijun) [reply docs/sdk/fragments/_get-user-information.mdx]: Update. Only add to those docs where related apis are available.

**Pull request merged**